app/src/main/java/com/hyperether/getgoing/ui/fragment/ShowDataFragment.java

```diff
@@ -122,7 +115,7 @@ public void onDestroy() {
     }
 
     private void initializeViewModel() {
-        routeViewModel = ViewModelProviders.of(getActivity()).get(RouteViewModel.class);
+        routeViewModel = new ViewModelProvider(this).get(RouteViewModel.class);
```

Comment:
@zukovicfilip is this only view which access to RouteViewModel?

Reply:
Yes